veg2hab/io/common.py

```diff
@@ -20,13 +20,13 @@ class AccessDBInputs(BaseModel):
     description: ClassVar[str] = "Inladen van vegkartering o.b.v. de digitale standaard"
 
     shapefile: str = Field(
-        description="Bestandslocatie van de vegetatiekartering",
+        description="Vegetatiekartering (geovectorbestand / shapefile)",
```

Comment:
Bestandslocatie hier weg omdat je over t algemeen een al in arcgis ingeladen laag selecteerd lijkt me?

Reply:
Nou, iemand probeerde hier de .mdb betand in te voeren. Dus het moet duidelijk zijn dat het gaat om de shapefile van de digitale kartering.

Al doe je dat ook maximaal 1 keer fout natuurlijk, en in arcgis is het inderdaad al een laag, dus hoef je als je wil niet meer een bestand te openen.

Reply:
Nee niet echt, als ze het zo duidelijk vinden vind ik t best